policyengine_us/variables/gov/states/ar/tax/income/deductions/itemized/ar_itemized_deductions.py

```diff
@@ -0,0 +1,85 @@
+from policyengine_us.model_api import *
+
+
+class ar_itemized_deductions(Variable):
+    value_type = float
+    entity = TaxUnit
+    label = "Arkansas itemized deductions"
+    unit = USD
+    definition_period = YEAR
+    reference = "https://www.dfa.arkansas.gov/images/uploads/incomeTaxOffice/2022_AR1000F_and_AR1000NR_Instructions.pdf#page=21"
+    defined_for = StateCode.AR
+
+    def formula(tax_unit, period, parameters):
+        person = tax_unit.members
+        year = period.start.year
+        p = parameters(period).gov.states.ar.tax.income.deductions.itemized
+        if year < 2018:
+            instant_str = f"2018-01-01"
+        else:
+            instant_str = f"{year}-01-01"
+        p_ded = parameters(instant_str).gov.irs.deductions
+        agi = tax_unit("ar_agi", period)
+        head = person("is_tax_unit_head", period)
+        person_agi = person("ar_agi_person", period)
+        total_person_agi = tax_unit.sum(person_agi * head)
+
+        # Less salt deduction
+        deductions = [
+            deduction
+            for deduction in p_ded.itemized_deductions
+            if deduction
+            not in [
+                "salt_deduction",
+                "medical_expense_deduction",
+            ]
+        ]
+        less_salt_deds = add(tax_unit, period, deductions)
+
+        # Real estate tax + Personal property tax
+        real_estate_deds = add(tax_unit, period, ["real_estate_taxes"])
+
+        # Post-secondary Education Tuition Deduction
+        tuition_deds = tax_unit(
+            "ar_post_secondary_education_tuition_deduction", period
+        )
+
+        # Limitation on several items
+        # Medical and Dental Expense
+        medical_expenses = add(tax_unit, period, ["medical_expense"])
+        if year >= 2017:
+            instant_str = f"2017-01-01"
+        else:
+            instant_str = f"2013-01-01"
+        p_medical = parameters(instant_str).gov.irs.deductions.itemized.medical
+        medical_deds = max_(
+            0,
+            medical_expenses - p_medical.floor * agi,
+        )
+
+        # Miscellaneous Deductions
+        misc_deds = tax_unit("misc_deduction", period)
+        adjusted_misc_deds = max_(
+            0,
+            misc_deds - p_ded.itemized.misc.floor * agi,
+        )
+
+        total_itemized_deduction = (
+            less_salt_deds
+            + medical_deds
+            + real_estate_deds
+            + tuition_deds
+            + adjusted_misc_deds
+        )
```

Comment:
make this a list parameter and use an adds function here
```suggestion
        total_itemized_deduction = (
            less_salt_deds
            + medical_deds
            + real_estate_deds
            + tuition_deds
            + adjusted_misc_deds
        )
```

Reply:
Just to make a clarification, do you mean by using 
`adds = [the list of itemized deductions]`
,or using 
`itemized_deductions = add(tax_unit, period, [the list of itemized deductions])`